src/models/validators/branchCollection.js

```diff
@@ -7,7 +7,7 @@ const branchCollectionValidator = (value, options = {}) => {
   const { items } = value
   const { validator } = options
 
-  if (!items || !items.length) return 'Collection is empty'
+  if (!items || !items.length) return null
```

Comment:
@suzubara This looks like similar pattern to something I commented on in https://github.com/18F/e-QIP-prototype/pull/1669 – what's the intent around these error strings as return values?

Reply:
@ryanhofdotgov see my comment: https://github.com/18F/e-QIP-prototype/pull/1669#discussion_r287362832

@christopherhuii what test case is this fixing? is a branch collection supposed to be valid if it's empty?

Reply:
Gotcha, this change has been reverted